custom_components/tuya_local/devices/madimack_elite_v3_heatpump_updated.yaml

```diff
@@ -0,0 +1,200 @@
+name: Madimack Elite V3 pool heatpump
+primary_entity:
+  entity: climate
+  dps:
+    - id: 101
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: auto
+              icon: "mdi:refresh-auto"
+              value: auto
+            - dps_val: cold
+              icon: "mdi:snowflake"
+              value: cool
+            - dps_val: heating
+              icon: "mdi:hot-tub"
+              value: heat
+    - id: 102
+      name: mode
+      type: string
+      hidden: true
+    - id: 104
+      name: temperature
+      type: integer
+      mapping:
+        - constraint: temperature_unit
+          conditions:
+            - dps_val: f
+              range:
+                min: 60
+                max: 104
+      range:
+        min: 18
+        max: 40
+    # ^ in reality, the range is different for cool and auto

```

Comment:
Since there are min_temperature and max_temperature dps, those will be used for the range in the UI.  The range here should be the min of the min, and max of the max, as this will be enforced on input at the integration level.